tldraw/apps/tldraw-logseq/src/components/ToolButton/ToolButton.tsx

```diff
@@ -24,13 +25,26 @@ export const ToolButton = observer(({ id, icon, title, ...props }: ToolButtonPro
   // Tool must exist
   const Tool = [...app.Tools, TLSelectTool, TLMoveTool]?.find(T => T.id === id)
 
-  const shortcut = ((Tool as any)['shortcut'] as string[])?.join(', ').toUpperCase()
+  const shortcuts = (Tool as any)['shortcut']
+
+  const tooltipContent = shortcuts ? (
+    <>
+      {tooltip}
+      <span className="ml-2 keyboard-shortcut">
+        {shortcuts
```

Comment:
The elements may need to have `key` props

Reply:
Handled. I also eliminated a dom validation warning about a nested buttons.